### PR TITLE
Updates for APM .NET Tracer v0.8

### PR DESCRIPTION
**Pull request by lucaspimentel**
### What does this PR do?

- APM > Languages > .NET
  - add new supported libraries
  - merge the two separate tables of supported libraries into one
- APM > Advanced Instrumentation
  - add .NET instructions to change Agent host and port
  - add .NET support for distributed tracing and sampling priority
  - improve .NET code samples slightly

### Motivation
We released [version 0.8](https://github.com/DataDog/dd-trace-dotnet/releases/tag/v0.8.0-beta) of the .NET Tracer 

### Preview links
https://docs-staging.datadoghq.com/lpimentel/apm-dotnet-0.8/tracing/languages/dotnet/
https://docs-staging.datadoghq.com/lpimentel/apm-dotnet-0.8/tracing/advanced_usage/?tab=net

### Additional Notes
N/A

### CODE REVIEW

File: content/en/tracing/advanced_usage/_index.md [hunk 1]
```diff
@@ -1577,9 +1590,10 @@ Once the sampling priority has been set, it cannot be changed. This is done auto
 {{% /tab %}}
 {{% tab ".NET" %}}
 
-Coming Soon. Reach out to [the Datadog support team][1] to be part of the beta.
+Priority sampling is enabled by default. The default sampler automatically assigns a value of `AutoReject` or `AutoKeep` to traces, depending on their service and volume.
+
+The ability to set a trace's sampling priority manually is coming soon.
```
Comment (cswatt) [inline content/en/tracing/advanced_usage/_index.md]: Can we get rid of this line completely? We try not to promise future things in the docs, they should reflect only the current state. (exceptions for things that are in beta, as beta sorta implies coming soon.)

File: content/en/tracing/languages/dotnet.md [hunk 2]
```diff
@@ -126,7 +126,7 @@ apk add libc6-compat
 
 **Note:** If your application runs on IIS and you used the MSI installer, you don't need to configure environment variables manually and you may skip this section.
 
-**Note:** The profiler tries to attach to _any_ .NET process that is started while these environment variables are set. You should limit profiling only to the applications that need to be traced. **Do not set these environment variables globally as this causes _all_ .NET processes on the host to be profiled.**
+**Note:** The .NET runtime will try to load a profiler into _any_ .NET process that is started while these environment variables are set. You should limit profiling only to the applications that need to be traced. **Do not set these environment variables globally as this causes _all_ .NET processes on the host to be profiled.**
```
Comment (cswatt) [inline content/en/tracing/languages/dotnet.md]:
```suggestion
**Note:** The .NET runtime tries to load a profiler into _any_ .NET process that is started while these environment variables are set. You should limit profiling only to the applications that need to be traced. **Do not set these environment variables globally as this causes _all_ .NET processes on the host to be profiled.**
```

File: content/en/tracing/advanced_usage/_index.md (hunk 1 above)
Comment (lucaspimentel) [reply content/en/tracing/advanced_usage/_index.md]: Removed in d309058af05290be586bd8191ef71c9934f07823.

File: content/en/tracing/languages/dotnet.md [hunk 3]
```diff
@@ -273,53 +273,50 @@ Environment Variable       | Description
...
-| .NET Framework | 4.5+     | Windows | Public Beta  |
-| .NET Core      | 2.0+     | Windows | Public Beta  |
-| .NET Core      | 2.0+     | Linux   | Public Beta  |
+| Runtime                | Versions | OS             | Support type |
+| ---------------------- | -------- | ---------------| ------------ |
+| .NET Framework         | 4.5+     | Windows        | Public Beta  |
+| .NET Core <sup>1</sup> | 2.0+     | Windows, Linux | Public Beta  |
 
-**Note**: Libraries that target .NET Standard 2.0 are supported when running on either .NET Framework 4.6.1+ or .NET Core 2.0+.
+<sup>1</sup> There was an issue in .NET Core versions 2.1.0, 2.1.1, and 2.1.2 that could prevent profilers from working correctly. This issue was fixed in .NET Core 2.1.3. See [this GitHub issue][9] for more details.
```
Comment (cswatt) [inline content/en/tracing/languages/dotnet.md]:
```suggestion
<sup>1</sup> There is an issue in .NET Core versions 2.1.0, 2.1.1, and 2.1.2 that can prevent profilers from working correctly. This issue is fixed in .NET Core 2.1.3. See [this GitHub issue][9] for more details.
```

These versions still have these issues, so we use present tense.

File: content/en/tracing/languages/dotnet.md [hunk 4]
```diff
@@ -273,53 +273,50 @@ Environment Variable       | Description
...
-| Redis         | `StackExchange.Redis`                     | 1.0.187+   | Public Beta   | `StackExchangeRedis` |
-| Redis         | `ServiceStack.Redis`                      | 4.0.48+    | Public Beta   | `ServiceStackRedis`  |
-| Elasticsearch | `NEST` / `Elasticsearch.Net`              | 6.0.0+     | Public Beta   | `ElasticsearchNet`   |
-| MongoDB       | `MongoDB.Driver` / `MongoDB.Driver.Core`  | 2.2.0+     | Public Beta   | `MongoDb`            |
+<sup>2</sup> The `AspNet` integration is not enabled by default in .NET Tracer 0.8. To enable it manually for testing purposes, see the [release notes for .NET Tracer 0.8][7].
 
-The ADO.NET integration tries to instrument **all** ADO.NET providers. Support was tested with SQL Server (`System.Data.SqlClient`) and PostgreSQL (`Npgsql`). Other providers (e.g. MySQL, SQLite, Oracle) might work, but have not been tested yet.
+<sup>3</sup> The ADO.NET integration tries to instrument **all** ADO.NET providers. Support was tested with SQL Server (`System.Data.SqlClient`) and PostgreSQL (`Npgsql`). Other providers (e.g. MySQL, SQLite, Oracle) might work, but have not been tested yet.
```
Comment (cswatt) [inline content/en/tracing/languages/dotnet.md]:
```suggestion
<sup>3</sup> The ADO.NET integration tries to instrument **all** ADO.NET providers. Support was tested with SQL Server (`System.Data.SqlClient`) and PostgreSQL (`Npgsql`). Other providers (e.g. MySQL, SQLite, Oracle) might work, but have not been tested.
```

**Pull request merged**